webdriver-spec.html

```diff
@@ -8739,6 +8741,77 @@
 </section> <!-- /Take Element Screenshot -->
 </section> <!-- /Screen Capture -->
 
+<section>
+<h2>Get Text Function</h2>
+
+<div class=note>
+<p>One of the major inputs to this specification was the Open
+ Source <a href="http://seleniumhq.org">Selenium</a> project. This was
+ in wide-spread use before this specification written, and as such had
+ set user expectations of how the <a>Get Element Text</a> command
+ should work. As such, the function presented here is known to be
+ flawed, but provides the best compatibility with existing users.
+
+<p>The function presented here is
+ the <a href="https://github.com/SeleniumHQ/selenium/blob/w3c-atoms/javascript/atoms/dom.js#L979">
```

Comment:
Should the link point to a commit instead of a remote branch name that could be deleted?

Example: 
https://github.com/vkatsikaros/webdriver/blob/issue_678/webdriver-spec.html#L6 (branch that used to exist on remote and was deleted now 404s)
https://github.com/vkatsikaros/webdriver/blob/master/webdriver-spec.html#L6 (branch that still exists 200s and shows content)